python/ray/_private/worker.py

```diff
@@ -554,6 +554,21 @@ def debugger_port(self):
         worker_id = self.core_worker.get_worker_id()
         return ray._private.state.get_worker_debugger_port(worker_id)
 
+    @property
+    def get_job_logging_config(self):
+        """Get the job's logging config for this worker"""
+        if not hasattr(self, "core_worker"):
+            return None
+        job_config = self.core_worker.get_job_config()
+        try:
+            logging_config = pickle.loads(job_config.serialized_py_logging_config)
+            assert logging_config is not None
+        except EOFError:
+            # If JobConfig's logging config is not set, serialized_py_logging_config
+            # will not be set, causing an EOFError to be raised when trying to unpickle.
```

Comment:
We can just check `if job_config.serialized_py_logging_config` before `pickle.loads`?

Reply:
The serialized value of `None` is not `None` from what I remember. Let me check again.

Reply:
The serialized value of `None` is `b''`. I checked the serialized value before deserializing.